source/_integrations/gotify.markdown

```diff
@@ -0,0 +1,99 @@
+---
+title: Gotify
+description: Instructions on how to integrate a self-hosted Gotify service with your Home Assistant notifications.
+ha_release: 2021.08
+ha_category:
+  - Notifications
+ha_domain: notify_events
+ha_codeowners:
+  - '@benjmarshall'
+ha_iot_class: Cloud Push
+ha_platforms:
+  - notify
+---
+
+## Description
+
+The Gotify service is a platform for the **notify** component.
+
+This platform allows you to use [Gotify](https://gotify.net/), an open-source, self-hosted, and simple to set-up push-message server to receive notifications from Home Assistant on your Android device.
+
+## Setting up
+
+To start getting notifications, you need to follow those simple steps:
+
+1. Setup a Gotify server by following the instructions on their [docs site](https://gotify.net/docs/install). You can run the server using docker or a native Go binary. You must make sure your Gotify server is accessible from the open web for this component to be able to communicate with it.
+2. When you first login to your Gotify server's WebUI setup a new user (unless you want to only use the default admin user), then logout and log back in as your new user.
+3. Add a new application to your Gotify server on the **Apps** tab. Name the new application **Home Assistant**. You can upload the [Home Assistant logo](https://github.com/home-assistant/brands/raw/master/core_integrations/_homeassistant/icon.png) in order to add badges to your notifications.
+4. When your application has been created Gotify will present you with a matching **token**. Copy this now as we will need it in the next step.
+5. Add the Gotify integration to your installation by adding the following to your `configuration.yaml` file:
+
+```yaml
+gotify:
+  token: YOUR_APP_TOKEN
```

Comment:
```suggestion
  url: YOUR_GOTIFY_URL
  token: YOUR_APP_TOKEN
```

Reply:
Spot on, i had left out the required URL. Thanks!

Reply:
Changes included in latest commit as I also needed to add URL item to configuration item table.